interface.html

```diff
@@ -100,6 +100,14 @@
         </div>
       </div>
     </div>
+    <div class="form-group">
+      <div class="col-sm-4 control-label">
+        <label for="chart_name">Chart name (for JS API)</label> <a href="https://developers.fliplet.com/API/components/charts.html">Learn more</a>
```

Comment:
I'm unsure of whether the "for JS API" note is needed. We don't have this note in the form builder.

Reply:
It's not optional in the form builder, so this needed an explanation for why it's optional. See https://weboo.atlassian.net/browse/ID-458?focusedCommentId=13434 and it has been confirmed with CS.